### ISSUE TITLE
[Tabs] mat-tab-header should horizontally scroll to the end of div.mat-tab-header-pagination

### ISSUE THREAD

**Issue by stagefright5** [angular/components#17155]
#### Reproduction

Use StackBlitz to reproduce your issue: https://stackblitz.com/angular/gopxbvgdpkv?file=app%2Ftab-group-dynamic-example.ts

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/15051871/65311033-277f7380-dbad-11e9-8cf8-6dafc035761f.gif)

Steps to reproduce:
1. Add tabs until the div.mat-tab-header-pagination element appear
2. Select the last tab
3. Start deleting tabs one by one

#### Expected Behavior

The tab header section should re-scroll horizontally to the end (that is, where it meets the paginator)

#### Actual Behavior

The tab header does not re-scroll.

#### Environment

- Angular: 8.1.4
- Browser(s): All modern browsers

**Comment by vbourdeix** [comment]
Agreed, on a mobile device with 3 tabs, the user has to manually click the > button in order for the scroll to reach the limit.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
